### ISSUE TITLE
Allow single operation calls in initializer of dcl statements

### ISSUE THREAD

**Issue by joey-coleman** [symphonytool/symphony#128]
> The following bug was [originally reported](https://sourceforge.net/p/compassresearch/bugs/128/) on Sourceforge by [joeycoleman](https://sourceforge.net/u/joeycoleman/profile/), 2013-10-15 12:21:51.861000:

For the example

```
(dcl x : int := value-expr @ P)
```

we also need to allow single operations in place of `value-expr`, i.e.

```
(dcl x : int := op1() @ P)
```

as a special form.  (This comes from PGL during the visit to York.)

**Comment by joey-coleman** [comment]
> [Comment](https://sourceforge.net/p/compassresearch/bugs/128/#3d63) by [joeycoleman](https://sourceforge.net/u/joeycoleman/profile/), 2013-10-16 12:05:30.035000:
- Description has changed:

Diff:

```

--- old
+++ new
@@ -10,4 +10,4 @@
 (dcl x : int := op1() @ P)
```

-as a special form.
+as a special form.  (This comes from PGL during the visit to York.)

```

- **assigned_to**: Joey Coleman --> Claus B. Nielsen
```

**Comment by joey-coleman** [comment]
> [Comment](https://sourceforge.net/p/compassresearch/bugs/128/#75d8) by [joeycoleman](https://sourceforge.net/u/joeycoleman/profile/), 2013-10-16 12:05:30.524000:

Ok, barring the statement/action rewrite that Kenneth and I have discussed, it looks to me like the parser creates an AST that is compatible with doing this.  To point, it creates AAssignmentDefinition nodes in what looks like the same manner as Overture does.

**Comment by joey-coleman** [comment]
> [Comment](https://sourceforge.net/p/compassresearch/bugs/128/#5197) by [lausdahl](https://sourceforge.net/u/lausdahl/profile/), 2013-11-04 14:00:44.671000:

This is fixed and tested in [354f1c265546609144c681ad37b8cfc0f7594237]

**Comment by joey-coleman** [comment]
> [Comment](https://sourceforge.net/p/compassresearch/bugs/128/#b6d3) by [lausdahl](https://sourceforge.net/u/lausdahl/profile/), 2013-11-04 14:00:55.621000:
- **status**: open --> closed
- **Module**:  --> unknown
